From e6b954be1150f7535b6971517eddd4e0fdf4cbed Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:18 -0400
Subject: [redhat] kabi: Adding symbol find_get_pages_tag (mm/filemap.c)

Message-id: <b0eb30c6a75512bd7129ca05819e6179b240d288.1410510773.git.poros@redhat.com>
Patchwork-id: 96090
O-Subject: [PATCH RHEL7.1 078/233] kabi: Adding symbol find_get_pages_tag (mm/filemap.c)
Bugzilla: 1071098
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Find and return pages that match tag.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/find_get_pages_tag
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/find_get_pages_tag
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/find_get_pages_tag

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/find_get_pages_tag b/redhat/kabi/kabi-module/kabi_ppc64/find_get_pages_tag
new file mode 100644
index 0000000..0d0afb2
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/find_get_pages_tag
@@ -0,0 +1,2 @@
+#1-
+0x171434d6 find_get_pages_tag vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/find_get_pages_tag b/redhat/kabi/kabi-module/kabi_s390x/find_get_pages_tag
new file mode 100644
index 0000000..1fda10a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/find_get_pages_tag
@@ -0,0 +1,2 @@
+#1-
+0x0d8dcd7a find_get_pages_tag vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/find_get_pages_tag b/redhat/kabi/kabi-module/kabi_x86_64/find_get_pages_tag
new file mode 100644
index 0000000..7dc3d82
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/find_get_pages_tag
@@ -0,0 +1,2 @@
+#1-
+0xcb73f0f8 find_get_pages_tag vmlinux EXPORT_SYMBOL
-- 
1.7.1